VideoLocker/src/main/java/org/edx/mobile/task/Task.java

```diff
@@ -21,19 +23,26 @@
 
 public abstract class Task<T> extends RoboAsyncTask<T> {
 
-    private ProgressBar progressBar;
+    public enum Type {
+        USER_INITIATED, LOADING_CACHED, LOADING_NON_CACHED
```

Comment:
There used to be a refresh error mentioned in the wiki, which seems to have been removed in the restructuring. I have posted a query on this in the wiki's comments section.